libs/design-tokens-ui/shared/README.md

```diff
@@ -0,0 +1,7 @@
+# design-tokens-ui-shared
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test design-tokens-ui-shared` to execute the unit tests.
```

Comment:
NIT: Maybe a little bit more descriptive and specific shot explanation about the use case